configure.ac

```diff
@@ -1880,6 +1880,48 @@ then
 fi
 
 
+# Maxim Integrated MAXQ10XX
+ENABLED_MAXQ10XX="no"
+maxqpartnumber=""
+AC_ARG_WITH([maxq10xx],
+    [AS_HELP_STRING([--with-maxq10xx=PART],[MAXQ10XX PART Number])],
+    [
+        AC_MSG_CHECKING([for maxq10xx])
+
+        # Read the part number
+        maxqpartnumber=$withval
+
+        if test "$maxqpartnumber" = "MAXQ1065"; then
+            LIB_STATIC_ADD="$LIB_STATIC_ADD lib/libmaxq1065_api.a"
+            AM_CFLAGS="$AM_CFLAGS -DWOLFSSL_MAXQ1065"
+            ENABLED_MAXQ10XX="yes"
+        elif test "$maxqpartnumber" = "MAXQ108x"; then
+            LIB_STATIC_ADD="$LIB_STATIC_ADD lib/libmaxq108x_api.a"
+            AM_CFLAGS="$AM_CFLAGS -DWOLFSSL_MAXQ108X"
+            ENABLED_MAXQ10XX="yes"
+        else
+            AC_MSG_ERROR([need a valid MAXQ part number])
+        fi
+
+        case $host_os in
+        mingw* | cygwin* | msys* | pw32* | os2* | cegcc*)
+                ENABLED_MAXQ10XX="yes"
+                LIB_ADD="$LIB_ADD -lcrypt32"
+                LIB_STATIC_ADD="lib/LibFT4222-64.dll lib/ftd2xx-64.dll $LIB_STATIC_ADD"
+                enable_shared=no
+                enable_static=yes
+                ENABLED_SINGLETHREADED=yes
```

Comment:
I don't understand the need to force single threaded? Again why isn't the caller using `--disable-shared` and `--enable-singlethreaded`?

Reply:
I agree.  this can be achieved via `configure` flags. I am removing: 
```
                enable_shared=no
                enable_static=yes
                ENABLED_SINGLETHREADED=yes
                AM_CFLAGS="$AM_CFLAGS -DSINGLE_THREADED"
```